frontend/src/components/service/Details.tsx

```diff
@@ -100,7 +101,14 @@ export default function ServiceDetails() {
                       },
                       {
                         label: t('translation|Addresses'),
-                        getter: endpoint => endpoint.getAddressesText(),
+                        getter: endpoint => (
+                          <Box display="flex" flexDirection="column">
+                            {endpoint.getAddresses().map((address: string) => (
+                              <ValueLabel>{address}</ValueLabel>
+                            ))}
+                          </Box>
+                        ),
+                        cellProps: { style: { width: '40%', maxWidth: '40%' } },
```

Comment:
I think we merged a commit where this line is discarded.

Reply:
rebased with main and the line is there still, manually removing it now